db/migrations/mysql/000076_upgrade_users_v6.6.down.sql

```diff
@@ -0,0 +1 @@
+-- It is intentionally to not have rollback for removing AcceptedTermsOfServiceId column.
```

Comment:
@isacikgoz Could you offer some advice on this? My understanding is that we want to have "down" operations anyway to be consistent,  even if they produce a technically incorrect schema.

Reply:
This comment should probably say `AcceptedServiceTermsId` to keep it consistent with the postgres "down"

Reply:
> My understanding is that we want to have "down" operations anyway to be consistent, even if they produce a technically incorrect schema.

@streamer45 I was mainly following the convention shown in https://github.com/mattermost/mattermost-server/blob/ff9e765eaa6706f134c8db2ba987117fe81a99e8/db/migrations/mysql/000074_upgrade_users_v6.3.down.sql#L1


> This comment should probably say `AcceptedServiceTermsId` to keep it consistent with the postgres "down"

@ashishbhate Thank you :+1:

Reply:
The idea is to make previous versions work when we do the down operation. Or, revert an incorrect operation that cause failures. In this case we are removing an incorrect column that supposed to be removed already. My thinking is it would be fine to do so.